packages/edit-post/src/components/header/index.js

```diff
@@ -189,7 +188,7 @@ function Header( {
 					className="edit-post-header__post-preview-button"
 					forceIsAutosaveable={ hasActiveMetaboxes }
 				/>
-				<ViewLink />
+				<PostViewLink />
```

Comment:
```suggestion
				<PostViewLink showIconLabels={ showIconLabels } />
```

It looks like the showIconLabels prop is not being passed. Without this, even if "Show button text labels" is enabled, a tooltip with the same text will be displayed when the button text is focused.

Reply:
oh yeah, forgot to pass this prop. Thanks for noticing.